pkg/apis/projectcalico/v3/packetcapture.go

```diff
@@ -60,6 +60,23 @@ type PacketCaptureSpec struct {
 	// The ordered set of filters applied to traffic captured from an interface.  Each rule contains a set of
 	// packet match criteria.
 	Filters []PacketCaptureRule `json:"filters,omitempty" validate:"omitempty,dive"`
+
+	// Coordinates the scheduled start/stop capturing traffic for a PacketCapture
+	// +optional
+	AutoTermination *PacketCaptureAutoTermination `json:"autoTermination,omitempty" validate:"omitempty"`
+}
+
+// PacketCaptureAutoTermination contains instructions when to start or stop
+// capturing traffic
+type PacketCaptureAutoTermination struct {
+
+	// Defines the start time of a PacketCapture
```

Comment:
We should cover the edge cases a bit more in this comment: maybe something like
```
Defines the start time from which this PacketCapture will capture packets.  If omitted or the value is in the past, the capture will start immediately.
```
and
```
Defines the end time at which this PacketCapture will stop capturing packets.  If omitted the capture will continue indefinitely.  If the value is changed to the past, capture will stop immediately.
```

Ideally we'd also specify what happens to previously captured files - if a capture is stopped (exceeds EndTime) and then starts again (exceeds StartTime), I'm guessing we'd start again; overwriting the first file, and then the second and so on?  If the times get changed a bit but it never stops and restarts, would we just continue (i.e. no going back to the first file or anything)?